cocoa/src/toga_cocoa/libs/appkit.py

```diff
@@ -624,7 +624,9 @@ class NSLineBreakMode(Enum):
 ######################################################################
 # NSRunLoop.h
 
-NSDefaultRunLoopMode = c_void_p.in_dll(appkit, "NSDefaultRunLoopMode")
+NSRunLoop = ObjCClass("NSRunLoop")
+NSRunLoop.declare_class_property("currentRunLoop")
+NSDefaultRunLoopMode = NSString(c_void_p.in_dll(appkit, "NSDefaultRunLoopMode"))
```

Comment:
Are these needed at runtime, or just for the probe? If it's just for the probe, we should probably declare them in the probe base class, rather than add the overhead at runtime.

Reply:
They're only used in the probe, so I've moved them to the Cocoa tests_backend.